inventory/group_vars/sampleserver.yml

```diff
@@ -2,9 +2,11 @@
 sample_config_user: fworchsample
 sample_config_user_home: "/home/{{ sample_config_user }}"
 import_sample_server: localhost
-sample_config_hostname: isosrv
-importer_hostname: localhost
+sample_config_hostname: isosrv #todo: delete? not used anywhere
+importer_hostname: localhost 
 sample_data_rate: medium
+sample_role_purpose: demo
+sample_fortigate_name: fortigate_demo
```

Comment:
isn't this the same name we already use for the demo system?

Reply:
This is the default for the original purpose of the role